lib/kino_flame/runner_cell.ex

```diff
@@ -126,6 +189,42 @@ defmodule KinoFLAME.RunnerCell do
 
     env = {:%{}, [], envs}
 
+    backend_ast =
+      quote do
+        {FLAME.FlyBackend,
+         [
+           unquote_splicing(specs_opts),
+           env: unquote(env)
+         ]}
+      end
+
+    to_quoted_pool(attrs, backend_ast)
+  end
+
+  defp to_quoted(%{"backend" => "k8s"} = attrs) do
+    runner_pod_tpl_a_sigil_y =
+      {:sigil_y, [delimiter: "\"\"\""],
+       [{:<<>>, [indentation: 2], [attrs["runner_pod_tpl"]]}, []]}
+
+    backend_ast = quote do: {FLAMEK8sBackend, runner_pod_tpl: runner_pod_tpl}
+    pool_ast = to_quoted_pool(attrs, backend_ast)
+
+    quote do
+      if match?({:error, _}, Code.ensure_loaded(FLAMEK8sBackend)) do
+        raise "You need to install `flame_k8s_backend` and `YamlElixir` as dependency."
+      end
```

Comment:
This is the code generated for the user, so we want to keep it clean. Instead of putting the check here, we should check it upfront and show a warning in the UI.

We have similar logic for missing dependencies in [Database connection cell](https://github.com/livebook-dev/kino_db/blob/bb6af5bc11fec657cc76c974e6684ae48ac314ae/lib/kino_db/connection_cell.ex#L443C8-L443C19).

Reply:
Nice, I was looking for sth like this.